app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -76,13 +76,14 @@ class FxaRepo(
          */
         object AuthenticatedNoProfile : AccountState() // Before the profile is fetched async
         object NeedsReauthentication : AccountState()
-        object NotAuthenticated : AccountState() // Initial state
+        object NotAuthenticated : AccountState()
+        object Default : AccountState() // Initial state
```

Comment:
nit: this comment is redundant to the code so I'd remove it.

Also, this state could be named, `InitialState` or `Initial`.